boot/bootdata/hivesft.inf

```diff
@@ -1653,7 +1653,7 @@ HKLM,"SOFTWARE\Microsoft\Ole","EnableRemoteConnect",0x00000000,"N"
 ; SvcHost services
 HKLM,"SOFTWARE\Microsoft\Windows NT\CurrentVersion\SvcHost",,0x00000012
 HKLM,"SOFTWARE\Microsoft\Windows NT\CurrentVersion\SvcHost","DcomLaunch",0x00010000,"PlugPlay"
-HKLM,"SOFTWARE\Microsoft\Windows NT\CurrentVersion\SvcHost","netsvcs",0x00010000,"DHCP","BITS","lanmanserver","lanmanworkstation","Schedule","Themes","winmgmt"
+HKLM,"SOFTWARE\Microsoft\Windows NT\CurrentVersion\SvcHost","netsvcs",0x00010000,"DHCP","BITS","lanmanserver","lanmanworkstation","Schedule","Themes","winmgmt","wuauserv"
```

Comment:
This should likely remain commented out until the service is actually used by something, as per my other comment.

Reply:
@ThFabba i fixed locally all comments except this. Without this string service does not start (failed with 1060). Is this block from merging?

Reply:
Not starting the service is exactly what I was going for! :) (hence less memory use/boot time)
However if apps depend on the service running and not just the files existing, then of course you'll have to leave it in here.

Reply:
Ok, let's leave it not starting for now. From times to times i tring to install .NET 4.5.1 :)